### ISSUE TITLE
JAWS doesn't read "Alt" when it is the first thing in a menu shortcut

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6456]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.910
    OS Version      : Windows
    R Version       : Any

### Steps to reproduce the problem
- Start JAWS 2020 (version 2020.2001.70 ILM).
- Alt+T to open the tools menu
- Arrow key to move down to "Keyboard Shortcuts Help"

### Describe the problem in detail
- JAWS reads "Keyboard shortcuts help, Shift+K"

### Describe the behavior you expected
- The shortcut is actually Alt+Shift+k
- This happens with any menu item whose shortcut starts with "Alt"

**Comment by gtritchie** [comment]
I've reported this to JAWS, and also a somewhat-vaguely related issue to Qt. https://bugreports.qt.io/browse/QTBUG-82838

**Comment by gtritchie** [comment]
This is a fairly serious issue for screen reader users; many key accessibility shortcuts start with Alt, making them not just undiscoverable but browsing the menus, but downright misleading!

**Comment by gtritchie** [comment]
Confirmed this is fixed by move to Electron. JAWS 2022.2202.38 correctly reads menu shortcuts that begin with <kbd>Alt</kbd>. 

Verified with Electron build 2022.06.0-daily+249 on Windows-11.
